### PR TITLE
Test installing Bloop on Azure and Github Actions

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,6 +6,6 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: olafurpg/setup-scala@v4
-    - run: sbt unit/test
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v4
+      - run: bin/travis/test.sh 'unit/test'
```
Comment (marek1840) [inline .github/workflows/ci.yml]: it's no longer only `travis`, right? ;)

**Comment by olafurpg** [comment]
Should we merge this PR with #982 ?

**Comment by marek1840** [comment]
> Should we merge this PR with #982 ?

I believe so, it builds on @sswistun-vl work.

**Comment by sswistun-vl** [comment]
Sbt command should be the last command - its status code signals if the tests passed or not

File: bin/travis/test.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+function bloop_version {
+  cat build.sbt | grep "val bloop" | sed 's|[^0-9.]||g'
+}
+
+wget -O bin/coursier https://git.io/coursier-cli && chmod +x bin/coursier \
+&& bin/coursier launch ch.epfl.scala:bloopgun-core_2.12:$(bloop_version) -- server &>bloop.log &
+
+sbt "$1"
```
Comment (sswistun-vl) [inline bin/travis/test.sh]: Sbt command should be the last command - its status code signals if the tests passed or not

**Comment by tgodzik** [comment]
I was just looking if we can make it work for every other test, we can merge it with @sswistun-vl PR

**Pull request closed without merging**